lib/reader.js

```diff
@@ -51,23 +30,22 @@ module.exports = async feeds => {
         'Expected at least 1 feed with more than one entry'
     );
 
-    const rows = getRows(feeds);
+    const opts = {
+        directory: await directory({ unsafeCleanup: true }),
+        ...options,
+    };
+
+    const b = browserify();
 
-    const browserifyPack = pack({ raw: true });
-    browserifyPack.pause();
+    for (const feed of feeds) {
+        const hash = await hashFeed(feed);
+        const root = join(opts.directory, hash);
 
-    await rows.reduce((acc, row) => {
-        acc.then(
-            () =>
-                new Promise(resolve => {
-                    browserifyPack.write(row, null, resolve);
-                })
-        );
-        return acc;
-    }, Promise.resolve());
+        await unpackFeed(root, feed);
```

Comment:
should we use `Promise.all` here to unpack in parallel?

Reply:
Yea, could do